File: html-css-js/01/js/main.js

// var allVars = {
// 	v1: 3.14,
// 	prenom: 'name\'s named',
// 	txt2: "still \"text\" 'text2'",
// 	typeoftxt2: "type of allVars",
// }

// if(true) {
// 	//
// } else if(false) {
// 	//
// } else {
// 	//
// }

// switch(allVars.length) {
// 	case 1:
// 		console.log("1");
// 		break;
// 	case 2:
// 		console.log("2");
// 		break;
// 	case 3:
// 		console.log("3");
// 		break;
// 	default:
// 		alert("def");
// 		break;
// }

// var test = false;
// while(test){
// 	console.log("");
// 	test = false;
// }

// var lien = document.querySelector('a');
// alert("1er line trouvé: " + lien.attributes);

var tabLiens = document.querySelectorAll('a');

console.log("[FOREACH]");
tabLiens.forEach(element => {
	console.log(element.innerHTML);
});
console.log("[//FOREACH]");

console.log("[FOR]");
for (var i = 0; i < tabLiens.length; i++) {
	const elem = tabLiens[i];
	console.log(elem.innerHTML);
}
console.log("[//FOR]");

File: html-css-js/01/dart/main.dart
import 'dart:html';
import 'dart:js';

void main() {

  //   int iter = 0;

  //   while (iter < 10) {
  //     print(iter);
  //     iter++;
  //   }

  //   for (var i = 0; i < 5; i++) {
  //     print("-" * (i + 5));
  //   }

  //   do {
  //     print("do while");
  //     iter++;
  //   } while(iter < 20);

  // var elem1 = querySelector('#t1');
  // // div.text = "TESSSSSSSSSSSSSSSSSTED";
  // elem1.appendText("text");


  // var dfName = context.callMethod('prompt',['Name ?','']);

  // window.alert("Hello " + dfName);

  // var ul = querySelectorAll('#numbs > li');
  // ul.forEach((elem) => window.alert(elem.innerHtml));

  // var aAc = querySelector('#accueil');
  // window.alert(aAc.attributes['href']);
  // aAc.attributes['href'] = '#';
  // window.alert(aAc.attributes['href']);

  querySelector('#playfield-h1')
    ..style.color = "cyan"
    ..style.fontSize = "100px";

  querySelector('#numbs')
    ..style.listStyleType = "upper-roman";




  void changeColorD(String color) => querySelector('#playfield-h1')
    ..style.color = color;

  querySelector('#playfield-h1').onClick.listen((e) => changeColorD('gold'));


}